MapboxNavigation/RouteVoiceController.swift

```diff
@@ -61,6 +54,24 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     
     /**
+     If true, a noise indicating the user is going to be rerouted will play prior to rerouting.
+     */
+    public var playRerouteSound = false
+
+    
+    /**
+     Sound to play prior to reroute. Inherits volume level from `volume`.
+     */
+    public var rerouteSound = NSDataAsset(name: "reroute-sound", bundle: Bundle.navigationUI)
```

Comment:
Changed it to bring your own audioplayer https://github.com/mapbox/mapbox-navigation-ios/pull/269/commits/6a636275305e6f70dfe92246dce4c81c4dce92ad